misc/dbt-materialize/tests/adapter/test_utils.py

```diff
@@ -61,3 +63,21 @@ def models(self):
 @pytest.mark.skip(reason="Materialize supports the list_agg() function")
 class TestListagg(BaseListagg):
     pass
+
+
+# Materialize implements `timestamp`, which is an equivalent abbreviation for
+# `timestamp without timezone` (as required by the SQL standard)
+# See https://www.postgresql.org/docs/current/datatype-datetime.html
+class TestCurrentTimestamps(BaseCurrentTimestamps):
+    @pytest.fixture(scope="class")
+    def expected_schema(self):
+        return {
+             "current_timestamp": "timestamp with time zone",
+             "current_timestamp_in_utc_backcompat": "timestamp",
```

Comment:
For clarity: the [base implementation](https://github.com/dbt-labs/dbt-core/blob/main/tests/adapter/dbt/tests/adapter/utils/test_current_timestamp.py) uses `timestamp without timezone`.